docs/dev/applications/h2network.md

```diff
@@ -6,6 +6,31 @@ prev_section: system
 permalink: /docs/dev/h2network/
 ---
 
+
+
+`H2Network` extends the `H2` / `H2GIS` geospatial database to provide graph routing functionalities. 
+
+`H2Network` is based on the [Java Network Analyzer](https://github.com/orbisgis/java-network-analyzer) (`JNA`) library. 
+
+JNA provides a collection of graph theory and social network analysis algorithms. These algorithms are implemented on mathematical graphs using the [JGraphT](https://jgrapht.org/) library.
+
```

Comment:
H2GIS offers a collection of SQL functions on top of the JNA.

Reply:
ok